book/website/collaboration/creating-agendas.md

```diff
@@ -0,0 +1,111 @@
+(cl-creating-agendas)=
+# Creating agendas
+
+In this section we will cover why you might want to use an agenda, how to structure the beginning, middle, and end of an agenda, as well as how you might choose to build off of that agenda for the next meeting.
+Meetings can provide the center of a community connection through shared work and networking.
+When you design an agenda for a meeting think about what you are giving to your attendees and what you are asking of them.
+
+Agendas, used well, can help make your meetings more productive, transparent, and connective.
+An agenda can help someone decide if a meeting is relevent for them or help them know what to prepare.
+During a meeting an agenda can be a written agreement on what you are spending time on as a group, showcasing what the group values enough to spend time on.
+Finally, if you combine an agenda with live notes, this can turn into a written record of what was agreed to or ideas that were explored that can be referred to in the future.
+
+## The beginning
+
+The begining of anything is critical.
+Beginnings of agendas are connecting points, bridges from our everyday live into our collaborative working spaces.
+The start of any meeting happens when the first person logs into the call or walks into the room.
+Often the first person to arrive can set the tone through their greetings of subsequent members.
+
+The begining of an agenda is commonly focused on adminstration and housekeeping tasks.
+There are several conventions you can build off of here.
+[Sociocracy](https://www.sociocracyforall.org/meeting-format/) uses ADMIN to open their meetings.
```

Comment:
This is a great format! I'm wondering if you could cite other frameworks (not describe them if you're not familiar of course), but rather give folks other options outside of the excellent one you describered for Sociocracy. We're all about plurality at The Turing Way - and this sounds like a great mode of allow people to choose what might work for them! For example, the RAPID framework, LEAN, etc. https://blog.pigeonholelive.com/how-to-make-meetings-more-effective

Reply:
What you have linked in here are less meeting structures and more collaboration arches in my mind. A lot of the team science literature focuses at this 'arch' level and I've seen less specifically about agenda construction. Long and short of it, this sociocracy guide has probably the most concrete guidance I've come across. I would love to see other people build off of this however!

TLDR I'm probably not the one to extend this but would be delighted to see other people build off if it here!